spring-integration-core/src/main/java/org/springframework/integration/store/MessageMetadata.java

```diff
@@ -29,10 +29,14 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private final UUID messageId;
+	private UUID messageId;
```

Comment:
No, i was just curious. . . primarily due to _if not final should it be volatile_ kind of thing. . .